src/main/Config.h

```diff
@@ -105,6 +105,9 @@ class Config : public std::enable_shared_from_this<Config>
     // this should only be enabled when testing as it's a security issue
     bool ALLOW_LOCALHOST_FOR_TESTING;
 
+    // Set to use config file values for genesis ledger
+    // not setable in config file - only tests are allowed to do this
+    bool USE_CONFIG_FOR_GENESIS;
```

Comment:
Small comment about style, but the whitespace here doesn't respect the convention used in the rest of this file.

Reply:
which whitespace?

Reply:
I realize my previous comment wasn't specific enough, sorry. I was referring to the lack of a blank line between lines 110 and 111. The rest of this file has (mostly) followed the convention of configuration description, configuration variable definition, blank line, repeat.